### ISSUE TITLE
[Tech Debt] Accessibility. Announce selection state for ListItems

### ISSUE THREAD

**Issue by Eism** [musescore/MuseScore#23106]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Bug description

The entire infrastructure is ready to solve this problem, but for some reason, screen readers do not announce the state of list items.

### Steps to reproduce

1. Go to the Score page
2. Open the Parts dialog (toolbar - Parts button)
3. Select the first item by pressing Space - the selected state is not announced

### MuseScore Version

4

### Regression

No.

### Operating system

any
